Core/DISET/private/BaseClient.py

```diff
@@ -400,7 +404,7 @@ def __checkThreadID(self):
     """
     if not self.__initStatus['OK']:
       return self.__initStatus
-    cThID = thread.get_ident()
+    cThID = threading.ident
```

Comment:
```
In [13]: threading.ident
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-13-ab6e47a0c388> in <module>()
----> 1 threading.ident

AttributeError: 'module' object has no attribute 'ident'
```

Reply:
fixed.

Reply:
threading does not have a get_ident call either :-/